frontend/src/pages/dataset/components/dataset-tabs/usage-tab/usage-tab.tsx

```diff
@@ -0,0 +1,12 @@
+import { useGetDatasetQueryStatsDailyQuery } from '@/store/features/datasets/datasets-api-slice';
+import { UsageChart } from './usage-chart';
+
+type Props = {
+    datasetId: string;
+};
+
+export function UsageTab({ datasetId }: Props) {
+    const { data, isLoading } = useGetDatasetQueryStatsDailyQuery(datasetId);
```

Comment:
Instead of querying here and passing it, I would just query the data inside of the `UsageChart`